plugins/gimp/file-jxl-save.cc

```diff
@@ -85,82 +85,77 @@ static bool gui_on_change_distance(GtkAdjustment* adj_dist,
       gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(toggle));
 
-  g_message("lossless = %d", jxl_save_opts.lossless);
-
   if (jxl_save_opts.lossless) {
     gimp_scale_entry_set_sensitive((GtkObject*)adj_distance, false);
     gimp_scale_entry_set_sensitive((GtkObject*)adj_quality, false);
     gtk_adjustment_set_value(adj_quality, 100.0f);
     gtk_adjustment_set_value(adj_distance, 0.0f);
 
     jxl_save_opts.distance = 0;
-    jxl_save_opts.update_quality();
+    jxl_save_opts.UpdateQuality();
 
-    gtk_adjustment_set_value(adj_effort, 3);
-    jxl_save_opts.encoding_effort = 3;
+    gtk_adjustment_set_value(adj_effort, 7);
   } else {
     gimp_scale_entry_set_sensitive((GtkObject*)adj_distance, true);
     gimp_scale_entry_set_sensitive((GtkObject*)adj_quality, true);
-    gtk_adjustment_set_value(adj_quality, 100.0f);
-    gtk_adjustment_set_value(adj_distance, 0.1f);
+    gtk_adjustment_set_value(adj_quality, 90.0f);
+    gtk_adjustment_set_value(adj_distance, 1.0f);
 
-    jxl_save_opts.distance = 0.1f;
-    jxl_save_opts.update_quality();
+    jxl_save_opts.distance = 1.0f;
+    jxl_save_opts.UpdateQuality();
 
-    gtk_adjustment_set_value(adj_effort, 7);
-    jxl_save_opts.encoding_effort = 7;
+    gtk_adjustment_set_value(adj_effort, 3);
   }
   return true;
 }
 
-static bool gui_on_change_codestream(GtkWidget* toggle) {
+static bool GuiOnChangeCodestream(GtkWidget* toggle) {
   jxl_save_opts.use_container =
       !gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(toggle));
   return true;
 }
 
-static bool gui_on_change_uses_original_profile(GtkWidget* toggle) {
+static bool GuiOnChangeUsesOriginalProfile(GtkWidget* toggle) {
   jxl_save_opts.basic_info.uses_original_profile =
       gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(toggle));
   return true;
 }
 
-static bool gui_on_change_advanced_mode(GtkWidget* toggle,
-                                        std::vector<GtkWidget*> advanced_opts) {
+static bool GuiOnChangeAdvancedMode(GtkWidget* toggle,
+                                    std::vector<GtkWidget*> advanced_opts) {
   jxl_save_opts.advanced_mode =
       gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(toggle));
 
   GtkWidget* frame = advanced_opts[0];
-
   gtk_widget_set_sensitive(frame, jxl_save_opts.advanced_mode);
 
   if (!jxl_save_opts.advanced_mode) {
-    jxl_save_opts.lossless = false;
-    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[1]), false);
-
-    jxl_save_opts.use_container = true;
-    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[2]), false);
-
     jxl_save_opts.basic_info.uses_original_profile = false;
-    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[3]), false);
+    gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[1]), false);
 
     // save metadata
-    // gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[4]), false);
+    // gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(advanced_opts[2]), false);
```

Comment:
Is this line meant to be uncommented, or is it not supposed to be here?

Reply:
It's meant to be commented, but I'll remove it entirely along with the "save metadata" placeholder option.